### PR TITLE
Feature/meal types

### PR DESCRIPTION
**Pull request by Tomekkk**
After merge deployment of backend script will be needed.

### CODE REVIEW

File: app/src/main/java/com/macoscope/ketchuplunch/model/lunch/MealService.kt [hunk 1]
```diff
@@ -15,10 +15,11 @@ class MealService(val scriptClient: ScriptClient, val userName: String) {
                 .filter { (it["name"] as String).isNotEmpty() }
                 .map {
                     it ->
-                    Meal(it["name"] as String, (it["count"] as BigDecimal).toInt(), (it["totalCount"] as BigDecimal).toInt())
+                    Meal(it["name"] as String, (it["count"] as BigDecimal).toInt(),
+                            (it["totalCount"] as BigDecimal).toInt(), it["type"] as String)
                 }
 
-        return meals
+        return meals.plus(meals)
```
Comment (dbacinski) [inline app/src/main/java/com/macoscope/ketchuplunch/model/lunch/MealService.kt]: what is this?

File: app/src/main/java/com/macoscope/ketchuplunch/view/lunch/LunchHeaderItemUI.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import org.jetbrains.anko.matchParent
+import org.jetbrains.anko.textView
+
+class LunchHeaderItemUI : AnkoComponent<ViewGroup> {
+    override fun createView(ui: AnkoContext<ViewGroup>): View {
+        return with(ui) {
+            frameLayout() {
+                lparams(width = matchParent, height = dip(28f))
+                backgroundColor = ContextCompat.getColor(ctx, R.color.lightGrey)
+                textView {
```
Comment (dbacinski) [inline app/src/main/java/com/macoscope/ketchuplunch/view/lunch/LunchHeaderItemUI.kt]: isn't it enough to use only TextView without wrapping?

**Comment by dbacinski** [comment]
-1

File: app/src/main/java/com/macoscope/ketchuplunch/model/lunch/MealService.kt (hunk 1 above)
Comment (Tomekkk) [inline app/src/main/java/com/macoscope/ketchuplunch/model/lunch/MealService.kt]: Debug code, I will delete it

File: app/src/main/java/com/macoscope/ketchuplunch/view/lunch/LunchHeaderItemUI.kt (hunk 2 above)
Comment (Tomekkk) [inline app/src/main/java/com/macoscope/ketchuplunch/view/lunch/LunchHeaderItemUI.kt]: I had problems with sticky headers with TextView, which widht was set to matchParent

**Comment by dbacinski** [comment]
+2

**Pull request merged**